pkg/proctree/events_processing.go

```diff
@@ -0,0 +1,37 @@
+package proctree
+
+import (
+	"github.com/aquasecurity/tracee/types/trace"
+)
+
+// ProcessEvent update the process tree according to arriving event
+func (tree *ProcessTree) ProcessEvent(traceeEvent *trace.Event) error {
+	switch traceeEvent.EventName {
+	case "sched_process_fork":
+		return tree.processForkEvent(traceeEvent)
+	case "sched_process_exec":
+		return tree.processExecEvent(traceeEvent)
+	case "sched_process_exit":
+		return tree.processExitEvent(traceeEvent)
+	case "exit", "init_namespaces":
+		return nil
+	default:
+		return tree.processDefaultEvent(traceeEvent)
+	}
+}
+
+// processDefaultEvent tries to expand the process tree in case of lost events or missing general information
```

Comment:
It is also called by processForkEvent, processExecEvent and processExitEvent, as a "baseline" for the process creation, right ? Maybe change the function doc to this "baseline" concept ? Maybe you meant it in the "missing general information" but it wasn't clear for me.